.github/workflows/ci.yml

```diff
@@ -61,6 +61,46 @@ jobs:
           path: .coverage.${{ matrix.python-version }}
           include-hidden-files: true
 
+  older-db-versions:
```

Comment:
I'm a bit weary of adding so many tests. What about changing the existing python matrix so that at the same time as we test old python versions, we test old postgres too ?

Reply:
My first version exactly did this, but I did not like it. I don't want to run coverage or upload artifacts. Of course, one can do this conditionally, but the job becomes more complicated. Also, the matrix itself becomes more complex as we want to test every Python version, but not every Python version against every database version. So, overall, I find a separate job more clear.

Reply:
So it would look [something like this](https://github.com/procrastinate-org/procrastinate/pull/1307/files) then. Do you think it's better? Or do you have something else in mind?
The `--cov` quite matters (running the whole test suite on my Laptop 47s vs 55s).

Reply:
@Xowap I understood ewjoachim wrong, but his suggestion makes total sense now. Would you mind adapting the CI code to something like this: https://github.com/procrastinate-org/procrastinate/pull/1307/files
Sorry for the extra work and that I didn't think of it myself 🤦.

Reply:
@Xowap I had some time left and quickly changed it 🙂.